docs/user-guide/api-mediation/api-mediation-multi-tenancy.md

```diff
@@ -33,96 +31,54 @@ The following diagram illustrates communication between the "central" API Mediat
-The Central API ML serves as a single point of access to all API Mediation Layers registered in it, and by extension, to all services registered in those secondary API MLs.
+The Central API ML serves as a single point of access to all API Mediation Layers registered in this Central API ML and, by extension, to all services registered in those secondary API MLs.
 
 Domain-1 to Domain-N are z/OS systems with the standard Zowe API ML running either in HA (sysplex) or non-HA (monoplex). These API MLs are referred to as Domain API MLs.
 
 ## Multitenancy component enablement settings
 
-In the multitenancy environment, certain Zowe components may be enabled, while others may be disabled. The multitenancy environment expects one Central API ML that handles the discovery and registration as well as routing to the API ML installed in specific domains. As such, different setups are required for the V2 version of the API ML on the central domain and on the specific customer environments. 
+In the multitenancy environment, certain Zowe components may be enabled, while others may be disabled. The multitenancy environment expects one Central API ML that handles the discovery and registration as well as routing to the API ML installed in specific domains. 
 
-When using a multitenancy environment, ensure that the following Zowe components are either enabled or disabled:
+## Onboarding a Domain Gateway service to the Central Discovery service
 
-- **Domain API ML**
-  - Gateway and Discovery Service: **enabled**
-  - Cloud Gateway: **disabled**
+The Central API ML can onboard Gateways of all domains. This service onboarding can be achieved similar to additional registrations of the Gateway. This section describes the dynamic configuration of the yaml file and environment variables, and how to validate successful configuration.
 
-- **Central API ML**
-  - Cloud Gateway and Discovery Service: **enabled**
-  - Gateway: **disabled**
-
-## Onboarding Domain Gateways to the Central Cloud Gateway
+- Dynamic configuration via zowe.yaml
+- Dynamic configuration via Environment variables
 
-The Central Cloud Gateway must onboard all Domain Gateways. This can be done dynamically or by static definition. We strongly recommend using dynamic onboarding as this onboarding method adapts better to the potentially changing environments of the customer. Static onboarding does not provide the functionality to actively monitor the health of specific services (e.g. domain gateways).  
+### Dynamic Configurations to the Central Discovery service
 
-### Dynamic Onboarding (recommended) for Domain Gateways
+#### Dynamic configuration: YML
 
-To dynamically onboard to the Discovery service in the central cluster, set the following property for all Domain Gateways:
+Set the following property for the Domain Gateway to dynamically onboard to the Central Discovery service.
```

Comment:
Is this step one and configuring Gateway to forward client certificates (optional) would be step two?